src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -61,6 +63,7 @@ public void InitialState(bool formatted, bool skipValidation)
             Assert.Equal(0, writer.BytesCommitted);
             Assert.Equal(0, writer.BytesPending);
             Assert.Equal(0, writer.CurrentDepth);
+            Assert.Equal(null, writer.Options.Encoder);
```

Comment:
Why not `Assert.Null(writer.Options.Encoder)`?

Reply:
Yes will fix. Thanks.